packages/grafana-ui/src/components/PanelChrome/PanelChrome.story.tsx

```diff
@@ -221,6 +222,7 @@ Basic.args = {
   title: 'Very long title that should get ellipsis when there is no more space',
   titleItems,
   menu,
+  loadingState: LoadingState.Loading,
```

Comment:
If we're deprecating `leftItems`, should we use `dataStateNode` + `loadingState` instead?

Reply:
Yes